### PR TITLE
Add disabled and sx props to MenuItemData

### PR DESCRIPTION
**Pull request by beaverusiv**
Fix docs for correct yarn command
Fix typo `nesteMenuItemsFromObjectProps`
Add `sx` and `disabled` as fields in `MenuItemData` to allow finer control from `menuItemsData` prop

### CODE REVIEW

File: packages/mui-nested-menu/src/components/IconMenuItem.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import MenuItem, { MenuItemProps } from '@mui/material/MenuItem';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/system/Box';
 import styled from '@mui/material/styles/styled';
+import { SxProps } from '@mui/system';
```
Comment (steviebaa) [inline packages/mui-nested-menu/src/components/IconMenuItem.tsx:6]: Can this be changed to full path import following this 
PR: https://github.com/steviebaa/mui-nested-menu/pull/18

File: packages/mui-nested-menu/src/definitions.ts [hunk 2]
```diff
@@ -1,8 +1,12 @@
+import { SxProps } from '@mui/system';
```
Comment (steviebaa) [inline packages/mui-nested-menu/src/definitions.ts:1]: same here

**Review by steviebaa — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a couple comments.

File: packages/mui-nested-menu/src/components/IconMenuItem.tsx (hunk 1 above)
Comment (beaverusiv) [reply packages/mui-nested-menu/src/components/IconMenuItem.tsx]: Changed to `'@mui/system/styleFunctionSx'`

File: packages/mui-nested-menu/src/definitions.ts (hunk 2 above)
Comment (beaverusiv) [reply packages/mui-nested-menu/src/definitions.ts]: Changed to `'@mui/system/styleFunctionSx'`

**Comment by steviebaa** [comment]
Thanks for contributing @beaverusiv 

Published to [NPM as version 3.1.0](https://www.npmjs.com/package/mui-nested-menu/v/3.1.0).

Although it is a breaking change, the fix is a simple type fix so not worth jumping a major for that.

**Comment by steviebaa** [comment]
Just the typo fix is breaking i mean.

**Pull request merged**